pages/common/xxd.md

```diff
@@ -1,23 +1,23 @@
 # xxd
 
-> Create a hexdump from a binary file or vice-versa.
+> Create a hexadecimal representation (hexdump) from a binary file, or vice-versa.
 
-- Generate a hexdump from a binary file and display the output to console:
+- Generate a hexdump from a binary file and display the output:
 
 `xxd {{input_file}}`
 
-- Send the hexdump output to a file:
+- Generate a hexdump from a binary file and save it to a text file:
 
 `xxd {{input_file}} {{output_file}}`
 
-- Format output in columns of 8 octets per line:
+- Display the output with 10 columns of one octet (byte) each:
 
-`xxd -c {{8}} {{input_file}}`
+`xxd -c {{10}} {{input_file}}`
```

Comment:
No, that's a good question, actually. I changed the number deliberately, because 8 could induce people I to thinking this has something to do with the number of bits in a byte/octet, which isn't the case.